content/blog/entries/smithsonian-unit-code-update/contents.lr

```diff
@@ -0,0 +1,62 @@
+title: Smithsonian Unit Code Update
+---
+categories:
+
+cc-catalog
+gsoc
+gsoc-2020
+---
+author: charini
+---
+series: gsoc-2020-cccatalog
+---
+pub_date: 2020-08-03
+---
+body:
+## Introduction
+The Creative Commons (CC) Catalog project collects and stores CC licensed images scattered across the internet, such
+that they can be made accessible to the general public via the [CC Search](https://ccsearch.creativecommons.org/)
+and [CC Catalog API](https://api.creativecommons.engineering/v1/) tools. Numerous information associated with each
+image, which help in the image search and categorisation process are  stored via CC Catalog in the CC database.
+
+In my [previous blog post](https://opensource.creativecommons.org/blog/entries/flickr-sub-provider-retrieval/)
```

Comment:
Please use relative links when possible.  (See other PRs for examples).  This will make the website more portable (and make running locally work properly)